server/vulnerabilities/oval/parsed/unix_unameTest_test.go

```diff
@@ -0,0 +1,38 @@
+package oval_parsed
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestEval(t *testing.T) {
+	utest := UnixUnameTest{
+		States: []ObjectStateString{
+			NewObjectStateString("less than", "0:5.15.0-1004"),
```

Comment:
What is the `0:` here for?

Reply:
this is the closest I can find to the definition of the [epoch](https://developer.bigfix.com/relevance/reference/debian-package-version-epoch.html).  It's handled in the `utils.Rpmvercmp` package.

Reply:
Should it be tested with and without the epoch?